### PR TITLE
[Pinta.Resources] Use symbolic tool icons.

### PR DESCRIPTION
**Pull request by jpobst**
Pinta's original icons don't work as well anymore in the era of dark themes and HiDPI screens.

Transitioning to symbolic SVG icons allow them to be displayed better at higher resolutions, and to be automatically colored to be viewable on dark themes.  This updates the icons for Pinta's toolbox.

toolbox

Unfortunately we don't have a contributing artist, so it's a combination of icons scavenged from appropriately licensed resources and programmer art!

### CODE REVIEW

File: Pinta.Resources/Pinta.Resources.csproj
Comment (cameronwhite) [inline Pinta.Resources/Pinta.Resources.csproj:18]: There probably also needs to be an update to `installer/linux/install.proj` to grab the svg icons during the `make install` step

File: Pinta.Resources/icons/hicolor/scalable/readme.md
Comment (cameronwhite) [inline Pinta.Resources/icons/hicolor/scalable/readme.md:9]: I think there are also some icon credits in the About dialog, so maybe these can be merged in there? I'm not sure if the existing credits are for some of the icons removed in this change :)

**Review by cameronwhite — COMMENTED** [review]
Nice! The new icons look pretty good to me
There were a few bugs on launchpad about this, so it's great to see this being worked on 👍 

I'm planning to do a 2.0.1 release imminently for some of the regressions that have been fixed, so I'll wait until after to merge this

File: Pinta.Resources/Pinta.Resources.csproj
Comment (jpobst) [reply Pinta.Resources/Pinta.Resources.csproj:18]: Good call, added!

File: Pinta.Resources/icons/hicolor/scalable/readme.md
Comment (jpobst) [reply Pinta.Resources/icons/hicolor/scalable/readme.md:9]: Added these sets to the existing list in the About box.  The remaining old icons we use are still a mix of all the existing sets, so I don't think we can remove those credits until all the `.png` files are replaced.

I also removed the "Powered by Mono" part of the About box while making this change, since that is no longer accurate.  😁

**Review by cameronwhite — APPROVED** [review]

**Comment by lobre** [comment]
Hi, not sure if my problem is related to this PR, but commenting just in case.

I am running Pinta 2.0.2 installed with nixpkgs on Ubuntu 20.04. When I open it, I receive this exception.

```
Icon 'tool-move' not present in theme Adwaita

GLib.GException: Icon 'tool-move' not present in theme Adwaita
   at Gtk.IconTheme.LoadIcon(String icon_name, Int32 size, IconLookupFlags flags)
   at Pinta.Core.GtkExtensions.LoadIcon(IconTheme theme, String icon_name, Int32 size) in /build/source/Pinta.Core/Extensions/GtkExtensions.cs:line 181
   at Pinta.Tools.MoveSelectedTool.get_DefaultCursor() in /build/source/Pinta.Tools/Tools/MoveSelectedTool.cs:line 47
   at Pinta.Core.BaseTool..ctor(IServiceManager services) in /build/source/Pinta.Core/Classes/BaseTool.cs:line 58
   at Pinta.Tools.BaseTransformTool..ctor(IServiceManager services) in /build/source/Pinta.Tools/Tools/BaseTransformTool.cs:line 47
   at Pinta.Tools.MoveSelectedTool..ctor(IServiceManager services) in /build/source/Pinta.Tools/Tools/MoveSelectedTool.cs:line 39
   at Pinta.Tools.CoreToolsExtension.Initialize() in /build/source/Pinta.Tools/CoreToolsExtension.cs:line 52
   at Pinta.MainWindow.OnActivated() in /build/source/Pinta/MainWindow.cs:line 103
   at GLib.Application.Activated_cb(IntPtr inst)
```

This happens whatever the GTK theme.
Could this be related to the switch to SVG icons?

Thanks

**Comment by cameronwhite** [comment]
It should be unrelated since this PR wasn't in the 2.0.2 release
That sounds like a packaging or path issue , where it can't find Pinta's custom icons that are installed under `<prefix>/share/icons/hicolor/...`

**Comment by lobre** [comment]
Thanks for the hint @cameronwhite and sorry for the noise then.

**Comment by Imerion** [comment]
Could there perhaps be an option for choosing the old icons instead? I prefer the high-colored look and I feel it is easier to quickly find the correct one with a bit of color. If not, perhaps they could be made available as a plugin.

**Comment by N0rdmann** [comment]
**+1** on Imerion's request
I regularly switch between paint.net and Pinta and would love to have back matching icons 🥰
_would be nice if this can be somehow archived without recompiling etc._

**Comment by cameronwhite** [comment]
Pull requests are welcome for this - I'm mainly focused on the GTK4 port currently 
It perhaps could be solved with changing the icon theme, but I'm not sure if it works to override symbolic icons with normal coloured icons

**Pull request merged**
